dm/worker/server.go

```diff
@@ -190,6 +199,18 @@ func (s *Server) Start() error {
 	return terror.ErrWorkerStartService.Delegate(err)
 }
 
+func (s *Server) startKeepAlive() {
+	defer s.wg.Done()
+	// worker keepalive with master
+	// If worker loses connect from master, it would stop all task and try to connect master again.
+	s.kpaCtx, s.kpaCancel = context.WithCancel(context.Background())
```

Comment:
we might use `s.ctx` rather than `context.Background()`. because line 163 doesn't have its corresponding `stopKeepAlive`, thus it may never quit.